src/tree/assess/scoreAggregators/SEOScoreAggregator.js

```diff
@@ -0,0 +1,54 @@
+import ScoreAggregator from "./ScoreAggregator";
+
+/**
+ * The number to scale the score to.
+ *
+ * Individual scores are from 1 to 9.
+ * The total score should be multiplied by this number to scale up.
+ *
+ * @type {number}
+ * @const
+ *
+ * @memberOf module:tree/assess
+ */
+const ScoreScale = 100;
+
+/**
+ * The factor to multiply the amount of results with.
+ *
+ * Individual scores are from 1 to 9.
+ * The make the total score work in the 100 scale, the amount of results needs to get multiplied by this factor.
+ *
+ * @type {number}
+ * @const
+ *
+ * @memberOf module:tree/assess
```

Comment:
I am not sure if this is needed, since it is a local variable that is not exported.

Reply:
As discussed with you. We document private things too. And even though this is not on the class directly, it is still a part of it.